src/components/PostStatusVisibility.js

```diff
@@ -38,6 +37,20 @@ class PostStatusVisibility extends React.Component {
         </div>
       )
     }
+
+    var postStat;
+    if (this.props.postStatus) {
+      postStat = (
+        this.props.postStatus
+      )
+    } else {
+      postStat = (
+        <div>
+          Not<span style={{display: "block"}}>Submitted</span>
+        </div>
```

Comment:
What is the purpose of this div? It might be simpler to just use ```defaultProps``` to set the default value of ```this.props.postStatus``` and then to simply use ```{this.props.postStatus}``` in the return method of render(). For info about ```defaultProps```, see [here](https://engineering.musefind.com/our-best-practices-for-writing-react-components-dec3eb5c3fc8).